src/vehicle.cpp

```diff
@@ -1248,6 +1252,152 @@ int vehicle::install_part( int dx, int dy, const vehicle_part &new_part )
     return parts.size() - 1;
 }
 
+bool vehicle::find_rackable_vehicle( std::vector<std::vector<int>> racks_parts )
+{
+    // the racks are already sorted
+    for( auto rack_parts : racks_parts ) {
```

Comment:
rack_part vs rack_parts vs racks_parts is very confusing, need these names to be a bit distinct.

Reply:
Fixed.